core/ui/TagTemplate.tid

```diff
@@ -13,6 +13,12 @@ color:$(foregroundColor)$;
 <$transclude tiddler={{!!icon}}/> <$view field="title" format="text" />
 </$button>
 <$reveal state=<<qualify "$:/state/popup/tag">> type="popup" position="below" animate="yes"><div class="tc-drop-down"><$transclude tiddler="$:/core/ui/ListItemTemplate"/>
+<$button class="tc-btn-invisible">
+<$list filter="[tag{!!title}]">
+<$action-navigate $to=<<currentTiddler>> />
+</$list>
+Open All
+</$button>
 <hr>
```

Comment:
Hi, I think it would be nice, to have a transclusion there. eg;  `{{$core/ui/TagFeatureButtons}}`. IMO this would allow us, to easily add more features to the tags dropdown, without modifying the template again. eg: have a "new here" button or "create new tiddler taged" ... and so on. We would just add a different `{{$core/ui/TagFeatureButtons}}`

I didn't test my proposal yet. .... but I think this may be handy.  what do you think?